src/util.js

```diff
@@ -1909,3 +1909,25 @@ export const sync = (dataUpdates={}, contextChildrenUpdates={}, { localOnly, for
     }
   }
 }
+
+/**
+ * Checks if str contains HTML elements
+ * @param {String} str 
+ */
+export const isHTML = (str) => /<\/?[a-z][\s\S]*>/i.test(str);
+
+/**
+ * Replaces certain reserved HTML chars
+ * @param {String} str 
+ */
+export const replaceSpecialHTMLChars = (str) => {
+  const rules = { '(&amp;|&)': 'and' }
+  let res = str
+
+  for (let rule of Object.keys(rules)) {
+    const regex = new RegExp(rule, 'gi')
+    res = res.replace(regex, rules[rule])
+  }
+
+  return res
+}
```

Comment:
Again note the Contributing Guidelines. `let` and `for` are not allowed on this project.